### PR TITLE
PDA-43: List BPM Groups

### PR DESCRIPTION
**Pull request by werbth**
Jira ticket: https://jira.entando.org/browse/PDA-43

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

File: src/main/java/org/entando/plugins/pda/controller/GroupController.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    private final ConnectionService connectionService;
+
+    private final EngineFactory engineFactory;
+
+    @Secured(GROUP_LIST)
+    @ApiOperation(notes = "Lists BPM groups", nickname = "listGroups", value = "LIST Group")
+    @GetMapping
+    public SimpleRestResponse<List<String>> listGroups(@PathVariable String connId,
+            @QueryParam("containerId") String containerId,
+            @QueryParam("processId") String processId) {
```
Comment (ffleandro) [inline src/main/java/org/entando/plugins/pda/controller/GroupController.java]: Should we use an id concatenated, same as we are using on Process Diagram endpoint?

**Review by ffleandro — CHANGES_REQUESTED** [review]

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Review by ffleandro — APPROVED** [review]

**Comment by ffleandro** [comment]
Retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by ffleandro** [comment]
Retest this please

**Comment by entando-service** [comment]
All is well

**Pull request merged**